tests/wdio.conf.js

```diff
@@ -12,6 +12,8 @@ const getBrowserLogFilePath = (specs) => {
   return path.join(__dirname, 'logs', 'browser.' + specName + '.log');
 };
 const browserLogPath = path.join(__dirname, 'logs', 'browser.console.log');
+const browserUtils = require('./utils/browser');
+const existingFeedBackDocIds = [];
```

Comment:
I think it would be better for this array to exist in the `utils/browser.js` file so it will not even write out the feedback docs. It also means if we use the function somewhere else we won't have to rewrite this bit. I'm thinking something like...

```js
const feedBackDocs = await browser.executeAsync(feedBackDocsScript);
const filtered = feedbackDocs.flat().filter(doc => return existingFeedBackDocIds.contains(doc.id));
```

NB: this is untested and probably wrong but you get the idea!

Reply:
I have updated to check the array within the function.